public/static/docs/user-guide/troubleshooting.md

```diff
@@ -3,8 +3,7 @@
 In this section we provide help for some of the problems that DVC user might
 stumble upon.
 
-<!-- Hardcoded id for error.dvc.org/many-files -> #many-files redirect -->
-<h2 id="many-files">Too many open files error</h2>
+## Too many open files error {#many-files}
```

Comment:
Oops, this is live now:

![image](https://user-images.githubusercontent.com/1477535/72760680-3730b800-3b9f-11ea-9a3c-76a1f5c54bea.png)
> https://dvc.org/doc/user-guide/troubleshooting

Reply:
Seems like it was a browser cache issue on my client side.